templates/home/index.html.twig

```diff
@@ -10,16 +10,16 @@
         {% for key, project in data %}
             {% for student in project.students %}
                 <li class="list-group-item">
-                    <a href="{{ path('student', {name: student, project: key })|escape }}"> {{ student }}</a>
-                    (<span class="badge">Mentorius</span> {{ project.mentors[0] }})
+                    <a href="{{ path('student', {name: student, project: key }) }}"> {{ student }}</a>
+                    (<span class="badge">Mentorius</span> {{ project.mentors | first }})
```

Comment:
Geriau jau parodyti visus mentorius, o ne tik pirmą.
Šiuo atveju duomenyse yra tik 1 mentorius. Bet realybėje, jei duomenys yra masyvas (_array_), tai jau signalas, kad `for` yra geriau nei `first`/`[0]`